pyproject.toml

```diff
@@ -18,11 +18,10 @@
 #
 # Release a new version:
 #
-#  - git tag VERSION
-#  - git push --tags
-#  - Remove untracked files/dirs: git clean -fdx
-#  - python -m build
-#  - twine upload dist/*
+#  - go to the GitHub release tab: https://github.com/python/pyperformance/releases
+#  - click "Draft a new release" and fill the contents
+#  - finally click the "Publish release" button! Done!
```

Comment:
Also, it might be nice to point to the workflow file here, so things seem less magical.

```suggestion
#  - finally click the "Publish release" button! Done!  (see .github/workflows/publish.yml for what gets run)
```